tests/tests_integration/test_api/test_datapoints.py

```diff
@@ -58,30 +68,32 @@
-            f"{prefix} 105: hourly values, 1969-10-01 - 1970-03-01, numeric",
-            f"{prefix} 106: every minute, 1969-12-31 - 1970-01-02, numeric",
-            f"{prefix} 107: every second, 1969-12-31 23:30:00 - 1970-01-01 00:30:00, numeric",
-            f"{prefix} 108: every millisecond, 1969-12-31 23:59:58.500 - 1970-01-01 00:00:01.500, numeric",
-            f"{prefix} 109: daily values, is_step=True, 1965-1975, numeric",
-            f"{prefix} 110: hourly values, is_step=True, 1969-10-01 - 1970-03-01, numeric",
-            f"{prefix} 111: every minute, is_step=True, 1969-12-31 - 1970-01-02, numeric",
-            f"{prefix} 112: every second, is_step=True, 1969-12-31 23:30:00 - 1970-01-01 00:30:00, numeric",
-            f"{prefix} 113: every millisecond, is_step=True, 1969-12-31 23:59:58.500 - 1970-01-01 00:00:01.500, numeric",
-            f"{prefix} 114: 1mill dps, random distribution, 1950-2020, numeric",
-            f"{prefix} 115: 1mill dps, random distribution, 1950-2020, string",
-            f"{prefix} 116: 5mill dps, 2k dps (.1s res) burst per day, 2000-01-01 12:00:00 - 2013-09-08 12:03:19.900, numeric",
+            f"{TEST_PREFIX}001: outside points, numeric",
+            f"{TEST_PREFIX}002: outside points, string",
+            *[f"{TEST_PREFIX}{i:03d}: weekly values, 1950-2000, numeric" for i in range(3, 54)],
+            *[f"{TEST_PREFIX}{i:03d}: weekly values, 1950-2000, string" for i in range(54, 104)],
+            f"{TEST_PREFIX}104: daily values, 1965-1975, numeric",
+            f"{TEST_PREFIX}105: hourly values, 1969-10-01 - 1970-03-01, numeric",
+            f"{TEST_PREFIX}106: every minute, 1969-12-31 - 1970-01-02, numeric",
+            f"{TEST_PREFIX}107: every second, 1969-12-31 23:30:00 - 1970-01-01 00:30:00, numeric",
+            f"{TEST_PREFIX}108: every millisecond, 1969-12-31 23:59:58.500 - 1970-01-01 00:00:01.500, numeric",
+            f"{TEST_PREFIX}109: daily values, is_step=True, 1965-1975, numeric",
+            f"{TEST_PREFIX}110: hourly values, is_step=True, 1969-10-01 - 1970-03-01, numeric",
+            f"{TEST_PREFIX}111: every minute, is_step=True, 1969-12-31 - 1970-01-02, numeric",
+            f"{TEST_PREFIX}112: every second, is_step=True, 1969-12-31 23:30:00 - 1970-01-01 00:30:00, numeric",
+            f"{TEST_PREFIX}113: every millisecond, is_step=True, 1969-12-31 23:59:58.500 - 1970-01-01 00:00:01.500, numeric",
+            f"{TEST_PREFIX}114: 1mill dps, random distribution, 1950-2020, numeric",
+            f"{TEST_PREFIX}115: 1mill dps, random distribution, 1950-2020, string",
+            f"{TEST_PREFIX}116: 5mill dps, 2k dps (.1s res) burst per day, 2000-01-01 12:00:00 - 2013-09-08 12:03:19.900, numeric",
+            f"{TEST_PREFIX}119: hourly normally distributed (0,1) data, 2020-2024 numeric",
+            f"{TEST_PREFIX}120: minute normally distributed (0,1) data, 2023-01-01 00:00:00 - 2023-12-31 23:59:59, numeric",
```

Comment:
I guess a good test time series would have e.g. only data during DST transitions (and oppositely maybe one year later)

Reply:
What about: 
* 121 some burst of datapoints on transition dates
* 122 Burst scattered on all dates, but the transition dates.